### PR TITLE
Nemwel Vet clinic Milestone 3 : Query multiple tables

### PR DESCRIPTION
**Pull request by Nemwel-Boniface**
## Hello reviewer(s) :wave: :wave: 

 _Since I made my commit messages as detailed as possible, in the PR description I will only highlight what I did_ 

- [x] Created two new tables for the project `owners` and the `species` table
- [x] Modified the animal's table id making it auto generated
- [x] Made use of the `JOIN` clause to help me write complex queries which combine data from multiple tables
- [x] populated the `species` and `owners` table inserting more information in the tables
- [x] Wrote more complex queries which involved the use of a combination of aggregate functions, and the `JOIN` clause to retrieve different information from the tables. eg `SELECT full_name, COUNT(owner_id) AS Total_animals FROM owners
JOIN animals ON  owners.id = animals.owner_id
GROUP BY full_name
ORDER BY COUNT(name) DESC;` lists all owners starting with the one with most animals
- [x] Updated the project's README.md file updating the documentation of the project

Thank you for taking the time to review my project :pray:

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -10,4 +10,70 @@ INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_k
...
+
+-- Blossom
+UPDATE animals SET owner_id = 4 WHERE name LIKE 'Blossom';
+
+-- Dean Winchester owns Angemon and Boarmon
+-- Angemon
+UPDATE animals SET owner_id = 5 WHERE name LIKE 'Angemon';
+
+-- Boarmon
+UPDATE animals SET owner_id = 5 WHERE name LIKE 'Bourman';
```
Comment (SajjadAhmad14) [inline data.sql]: Please correct the typo here. It should be `Boarmon` instead of `Bourman`.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Nemwel, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: data.sql (hunk 1 above)
Comment (Nemwel-Boniface) [reply data.sql]: Thank you @SajjadAhmad14 for this review.
You are very keen thank you I will change the typo right away!

**Review by mateo951 — APPROVED** [review]
### Hi @Nemwel-Boniface, 

Your project is complete!  There is nothing else to say other than... it's time to merge it ✔ 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**